contracts/foundry-tests/LidoOracle.t.sol

```diff
@@ -0,0 +1,106 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.6;
+
+import "@yield-protocol/utils-v2/contracts/access/AccessControl.sol";
+import "../oracles/chainlink/ChainlinkMultiOracle.sol";
+import "../oracles/composite/CompositeMultiOracle.sol";
+import "../oracles/lido/LidoOracle.sol";
+import "../mocks/oracles/chainlink/ChainlinkAggregatorV3Mock.sol";
+import "../mocks/oracles/lido/WstETHMock.sol";
+import "../mocks/ERC20Mock.sol";
+import "../mocks/USDCMock.sol";
+import "../mocks/WETH9Mock.sol";
+import "./utils/Test.sol";
+import "./utils/TestConstants.sol";
+
+contract LidoOracleTest is Test, TestConstants, AccessControl {
+    WETH9Mock public weth;
+    ERC20Mock public steth;
+    USDCMock public usdc;
+    LidoOracle public lidoOracle;
+    WstETHMock public lidoMock;
+    ChainlinkMultiOracle public chainlinkMultiOracle;
+    CompositeMultiOracle public compositeMultiOracle;
+    ChainlinkAggregatorV3Mock public stethEthAggregator;
+    ChainlinkAggregatorV3Mock public usdcEthAggregator;
+
+    bytes6 public mockBytes6 = 0xd1eaa762fae7;
+
+    function setUp() public {
+        lidoMock = new WstETHMock();
+        lidoMock.set(1008339308050006006);
```

Comment:
These magic numbers should be extracted to variables.

```
uint256 stEthToWstEthPrice = 1008339308050006006
```

A comment should be also added to remind reviewers that `stEthToWstEthPrice` is the amount of wstEth that you get for 1e18 stEth.